### DISCUSSION TITLE
Problem with GET method /{content-type}/:id to get one document by id of a collection

### DISCUSSION THREAD

**Discussion by EmmanuelleZenou** [strapi/strapi#7624] [Legacy Topics]
Hello everyone,

I am new in Strapi and it is very simple :
 - I can get the information of a my collection Posts with the url http://localhost:1337/posts
returning :     [{"_id":"5ed560390b5bdce7318621c4","title":"Post1", blablabla .........}]
-but when I try http://localhost:1337/posts/5ed560390b5bdce7318621c4 to find only one document by id of the collection I have a "Not Found" error.

Can u please help me ?

Thank you !!!!!!

**Comment by derrickmehaffy** [comment]
Hi @EmmanuelleZenou I'm not able to reproduce this locally, can you please confirm what Strapi version you are using and have you customized the controllers or routes at all?

**Reply by EmmanuelleZenou** [reply]
Ok I use 3.1.4, I 've just discovered I had to customize the parameters my routes ^^' I thought it would work automatically just as in this tutorial :https://strapi.io/documentation/v3.x/content-api/api-endpoints.html  It is working now thank you very much !!!
